pkg/helm/helm.go

```diff
@@ -61,6 +64,7 @@ type Interface interface {
 	ChartLoad(ref string) (*chart.Chart, error)
 	ChartPush(chart *chart.Chart, ref string) error
 	ChartPull(ref string) error
+	GetResources(kubeNamespace string, releaseName string) ([]*unstructured.Unstructured, error)
```

Comment:
you have added a new call GetResources but you did not extend the unit test. Please do.

Reply:
Added testing in https://github.com/IBM/the-mesh-for-data/pull/53